### PR TITLE
Update for 2.1 

### PR DESCRIPTION
**Pull request by geggleto**
Ref #16

### CODE REVIEW

**Comment by geggleto** [comment]
ping @akrabat

File: README.md [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
+<<<<<<< HEAD
+## Templates
+You may use `$this` inside your php templates. `$this` will be the actual PhpRenderer object will allow you to render sub-templates
+=======
 ## Installation
 
 Install with [Composer](http://getcomposer.org):
 
     composer require slim/php-view
+>>>>>>> f5134be6ea4dac95e14439ce203927d57760ccf4
```
Comment (akrabat) [inline README.md]: Conflict messages need removing

**Comment by geggleto** [comment]
Ah, right.

**Comment by geggleto** [comment]
@akrabat ping

**Pull request merged**